### ISSUE TITLE
Problem when backing up the transmitter via the Suite

### ISSUE THREAD

**Issue by Wolepo** [FrSkyRC/ETHOS-Feedback-Community#4710]
Hello, I have backed up my transmitter completely using Suite, including the audio files. In the middle of my work, Suite cancelled with an error message. A 2nd attempt also failed. So I did it again manually. It turned out that a sound file was faulty.
Wouldn't it be an advantage to display such a faulty file and offer a selection to skip it? A path for a log is given, but I can't find it

**Comment by Nicholas-Luoyi** [comment]
Would you please try to backup a copy without all the audio files to see if it goes through the all way?

**Comment by Wolepo** [comment]
Yes, a backup worked without any problems and ran through. There is currently a version of Bertrand (1.6.0, X10S Express) on the transmitter.

**Comment by Wolepo** [comment]
I found the path to the first protocol after another attempt with the same result.
[main-20241109.log](https://github.com/user-attachments/files/17731005/main-20241109.log)

**Comment by Nicholas-Luoyi** [comment]
Do you have a log file starts with "crash-"?
